common/compositor/factory.cpp

```diff
@@ -17,7 +17,9 @@
 #include "factory.h"
 #include "platformdefines.h"
 
-#ifdef USE_GL
+#ifdef USE_DC
+#include "nativesurface.h"
```

Comment:
CreateMediaRenderer still returns VARenderer. 3D Renderer returns NULL.Did we miss changes in factory class ?